### PR TITLE
Update intro and find command

### PR DESCRIPTION
**Pull request by yongmingyang**
- Updated the intro & welcome message

- Updated the find command with tip as well as images for clarification purposes

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -108,18 +113,28 @@ If you would like to search the database, you can do so in a few ways:
...
+
+How it should look like in the application:
+
+![FindContext](images/FindContext.PNG)
+
+Expected Result:
+
+![FindResult](images/FindResult.PNG)
 
+The input `find t/Chronic diseases, Diagnosis` will filter out the questions with the tag "Chronic Diseases" and/or "Diagnosis"
```
Comment (jonfoocy) [inline docs/UserGuide.md]: "filter out" sounds like they're being omitted from the search? Maybe can change to "will return questions"

**Review by jonfoocy — APPROVED** [review]
Changes look good, just a minor change to consider making

File: docs/UserGuide.md (hunk 1 above)
Comment (yongmingyang) [reply docs/UserGuide.md]: aight changed

**Review by ChengJiyuqing — APPROVED** [review]
look good!

**Pull request merged**